### ISSUE TITLE
select 12/24 hour format independent of locale settings

### ISSUE THREAD

**Issue by CodeShredder726b** [kimai/kimai#3934]
### Describe the problem

I would like to set the hour format independently of the language/location setting. 

### Describe the solution you'd like

Add an additional setting option in the user preferences:
- Timezone
- Language
- **Time Format**
    **12h(AM/PM) and 24h**
- First day of the week

### Describe alternatives you've considered

As for now I set a wrong location but it does support the 24h because hour format is more important than location for me. But it is only a workaround.

**Comment by kevinpapst** [comment]
That setting was removed on purpose.
Which locale do you currently use and which would you use, if this setting would exist?

**Comment by freelancer1845** [comment]
Same here, I'm used to having everything in english, so I choose english as language. But I'm actually from germany so I'm not used to AM/PM format. 

I would use English (United States) and 24 hours format

I do understand though that this might seem a little strange...

I'm running version 2.0.10

**Comment by kevinpapst** [comment]
> I would use English (United States) and 24 hours format

Why not using English (German) English (United KIngdom) or any other one that uses 24 hours?
Why not use English (Maldives) which uses the exact same formats, except that the hours are 24.
Did you follow the link under the settings, which list all available locales with their settings?

<img width="743" alt="Bildschirm­foto 2023-03-20 um 19 02 57" src="https://user-images.githubusercontent.com/533162/226427746-c8900b4f-3ac4-4706-b4ab-cbfec35a6f16.png">

**Comment by freelancer1845** [comment]
Ah I just didn't realize that that is a link to something (of course I could have guessed from hovering or the color...) and I didn't know much about any locales, but really nice table :)

Thanks for the info 👍 

Maybe it's just me not discovering this as a link (: or there is a way to make it more clearer. I think I just thought it is the color of hints or something. Maybe only part of the text could be the link such that there is a clear difference in styling? And/Or the text reflects that this is something leading to a table of locales and how they play out

**Comment by CodeShredder726b** [comment]
I don't see any link under System->Settings->User->Language. 
![image](https://user-images.githubusercontent.com/16302796/231691546-607f011c-5fb5-4ef5-8725-e684289a5eac.png)

So it is Locale not just Language? Then it makes much more sense.

**Comment by kevinpapst** [comment]
99% of all users do not know what a Locale is, so "language" makes more sense as label.
Your version might not have the link, the latest tag does.

**Comment by hrvoj3e** [comment]
It does not work. I am using cloud version.
Used "English (Slovenia)" in profile settings and have selected  "English (Slovenia)" in Administration > Settings.

I also tried to clear all caches, local/session storage in browser.  No change...

New timesheets and existing timesheets are using AM/PM - it is so frustrating.

**Comment by kevinpapst** [comment]
Why would `System->Settings->User->Language` change anything for your session?
It clearly says `Default values for new entries`:
<img width="255" alt="Bildschirmfoto 2023-08-08 um 09 50 56" src="https://github.com/kimai/kimai/assets/533162/743054a4-ad8d-4e27-8981-10f5d7e02dd2">

Go your your own user preferences
<img width="374" alt="Bildschirmfoto 2023-08-08 um 09 51 41" src="https://github.com/kimai/kimai/assets/533162/682ae2d5-83ac-4029-951c-8a6e09fde815">

<img width="973" alt="Bildschirmfoto 2023-08-08 um 09 51 57" src="https://github.com/kimai/kimai/assets/533162/75a7e322-f606-433e-bb5b-5fff873d2b5d">

**Comment by hrvoj3e** [comment]
I used User > Preferences first. 

![image](https://github.com/kimai/kimai/assets/4988133/c4e34f33-c631-4080-9ea8-b53670212539)

When that did not change default times to 24h format, I used System > Setting > Language because "Google said so somewhere" - sorry.

I tried to log out again after some time (2-3 hours) and log in again and 24h format is visible now. Old time entries are using 24h also.
It works OK.

But I agree with the request. UI language and local should be separate - at least for time, date, currency format.
